src/scrconf/slp_conf.scr

```diff
@@ -37,7 +37,7 @@
         // we need to exclude ; because of the second matching rule
         "params" : [
                 // Options with one value ('yes' / 'no')
-                $[ "match" : [ "^[#;][ \t]*([^ \t\=]+)[ \t\=]+[ ]*(.+)[ \t]*$", ";%s = %s" ]],
+                $[ "match" : [ "^[#;][ \t]*([^ \t\=]+)[ \t\=]*[ ]*(.+)[ \t]*$", ";%s = %s" ]],
                 // Options with more possible values
```

Comment:
Now, before this fix, this 1st element of "params" differed from the 2nd only by the comment character: `^[#;]...` vs `^...`. Shouldn't the 2nd one be changed too?